packages/ag-charts-community/src/chart/interaction/regionManager.ts

```diff
@@ -67,7 +67,7 @@ function addHandler<T extends RegionEvent['type']>(
 
 export class RegionManager {
     private currentTabIndex = 0;
-    private readonly focusIndicator: HTMLElement;
+    public readonly focusIndicator: FocusIndicator;
```

Comment:
I am unsure where the ownership of the `FocusIndicator` object should be. Currently I'm thinking of these three possible options:

1) Keeping it here in `RegionManager`
2) Moving it to `ChartContext` passing the `FocusIndicator` object to `RegionManager`'s constructor
3) Moving it to `DOMManager` (and keeping `RegionManager`'s `domManager` constructor param as `private readonly`).

I'm leaning towards options 3), but it will likely change the `appendChild` ordering of the DOM elements.

Reply:
🤔 I would probably lean toward 2) as the FocusIndicator doesn't seem like a responsibility suitable for the `DOMManager`?

Reply:
ACK. I'll move it in a follow-up PR.